src/Utilities/Helpers.js

```diff
@@ -19,6 +19,11 @@ import {
 import { intl } from './IntlProvider';
 import { packageSystemsColumns } from '../SmartComponents/Systems/SystemsListAssets';
 
+export const deleteUnsetObjectValues = (object) => {
+    Object.keys(object).forEach(key => object[key] === undefined && delete object[key]);
```

Comment:
`delete object[key]` modifies the original object, it might be more useful if the original object remains unchanged (create a copy and delete it there). Just an idea that would make our debugging easier.